components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -245,6 +241,12 @@ - (void)layoutSubviews {
       self.floatingButton.center =
```

Comment:
looks like we need to fix the indentation here

Reply:
Fixed.